doc/classes/TileMap.xml

```diff
@@ -4,7 +4,7 @@
 		Node for 2D tile-based maps.
 	</brief_description>
 	<description>
-		Node for 2D tile-based maps. Tilemaps use a [TileSet] which contain a list of tiles which are used to create grid-based maps. A TileMap may have several layers, layouting tiles on top of each other.
+		Node for 2D tile-based maps. Tilemaps use a [TileSet] which contain a list of tiles which are used to create grid-based maps. A TileMap may have several layers layering tiles on top of each other.
```

Comment:
What about "laying out"? This would be the correct form of "layouting" while avoiding repetition.

Reply:
Laying out or "layout" typically refers to an arrangement of items in close proximity, not stacked like in this situation. It might be better to simplify and say that a Tilemap may contain several layers of tiles on top of each other.